### ISSUE TITLE
Visual Referee Challenge

### ISSUE THREAD

**Issue by ahasselbring** [RoboCup-SPL/Rules#111]
Thomas Röfer in Discord:
> I think it is problematic that robots should look at the referee during kick-off at least until five seconds after the whistle. They are supposed to play soccer at that point. This is a situation in which it is completely clear what the whistle means, so there would be no need to look at the referee in future matches in which the referee signs would have an actual meaning for the game. It is also problematic to detect a referee sign after the referee whistled to finish the half, because according to the rulebook, "The robots then have to sit down.". Traditionally, all robots stop immediately when the GameController sends the "finished" signal. For this challenge, they would have to turn around to see the referee.
In general, this challenge asks teams to risk their success in the soccer games for a potential success in the challenge (the "Data Minimization Challenge" does this as well). I don't think that this is a good approach to make challenges more popular.
Can you also please clarify "If multiple messages arrive only the first one from the robot with the lowest number, will be counted."? If number 2 send a message after 1 second and number 1 sends one after 14 seconds, the only message by number 1 is counted?
There is also an artifact from last year left in the description: "Only Messages with audio output and message sent within 15 second after the signalling started, will be accepted."

tl;dr:
- signals at kick-off are impractical and an unrealistic setting in real games
- signals at end-of-half require that players are allowed to move in finished state
- why should the player number matter in the result reporting?
- reference to audio output must be removed
- "risk [...] success in [...] soccer games for [...] success in the challenge" -> yes, but we are probably already past that decision

**Comment by ahasselbring** [comment]
- In the formula $sum_Time$, $P_goal$ and $P_end_of_half$ appear (should be $T$)
- "Player exchange" should be called "Substitution"

**Comment by PatrickGoettsch** [comment]
> * In the formula ∑Time, Pgoal and Pendofhalf appear (should be T)
> * "Player exchange" should be called "Substitution"

Have been addressed.

**Comment by PatrickGoettsch** [comment]
> Thomas Röfer in Discord:
> 
> > I think it is problematic that robots should look at the referee during kick-off at least until five seconds after the whistle. They are supposed to play soccer at that point. This is a situation in which it is completely clear what the whistle means, so there would be no need to look at the referee in future matches in which the referee signs would have an actual meaning for the game. It is also problematic to detect a referee sign after the referee whistled to finish the half, because according to the rulebook, "The robots then have to sit down.". Traditionally, all robots stop immediately when the GameController sends the "finished" signal. For this challenge, they would have to turn around to see the referee.
> > In general, this challenge asks teams to risk their success in the soccer games for a potential success in the challenge (the "Data Minimization Challenge" does this as well). I don't think that this is a good approach to make challenges more popular.
> > Can you also please clarify "If multiple messages arrive only the first one from the robot with the lowest number, will be counted."? If number 2 send a message after 1 second and number 1 sends one after 14 seconds, the only message by number 1 is counted?
> > There is also an artifact from last year left in the description: "Only Messages with audio output and message sent within 15 second after the signalling started, will be accepted."
> 
> tl;dr:
> 
> * signals at kick-off are impractical and an unrealistic setting in real games
> * signals at end-of-half require that players are allowed to move in finished state
> * why should the player number matter in the result reporting?
> * reference to audio output must be removed
> * "risk [...] success in [...] soccer games for [...] success in the challenge" -> yes, but we are probably already past that decision

1. That is later also the problem. When you think the referee whistles you would have to look to him. Which might be problematic in same game situations. But we have to learn to deal with that. We have 5 to 7 robots.
2. Robots are allowed to move that situation.
3. Is removed but you have to deal with what is the final answer in case you have multiple once.
4. Audio is removed
5. See 1.
